<ui version="4.0" >
 <class>Dialog_levelset_para</class>
 <widget class="QDialog" name="Dialog_levelset_para" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Define levelset parameters</string>
  </property>
  <widget class="QWidget" name="" >
   <property name="geometry" >
    <rect>
     <x>21</x>
     <y>24</y>
     <width>401</width>
     <height>300</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2" >
    <item>
     <layout class="QGridLayout" name="gridLayout_3" >
      <item row="0" column="0" >
       <layout class="QGridLayout" name="gridLayout_2" >
        <item row="0" column="0" >
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>Channel</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QSpinBox" name="spinBox_Channel" />
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>Segmentation</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QComboBox" name="comboBox_Dimension" >
          <item>
           <property name="text" >
            <string>2D</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>3D</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QLabel" name="label_3" >
          <property name="text" >
           <string>Starting slice</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <widget class="QSpinBox" name="spinBox_StartSlice" />
        </item>
        <item row="3" column="0" >
         <widget class="QLabel" name="label_4" >
          <property name="text" >
           <string>Ending slice</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" >
         <widget class="QSpinBox" name="spinBox_EndSlice" />
        </item>
       </layout>
      </item>
      <item row="0" column="1" >
       <layout class="QGridLayout" name="gridLayout" >
        <item row="0" column="0" >
         <widget class="QLabel" name="label_8" >
          <property name="text" >
           <string>Length weight (lamda)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QDoubleSpinBox" name="doubleSpinBox_lamda" />
        </item>
        <item row="2" column="0" >
         <widget class="QLabel" name="label_9" >
          <property name="text" >
           <string>Area weight (alpha)</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <widget class="QDoubleSpinBox" name="doubleSpinBox_alpha" />
        </item>
        <item row="3" column="0" >
         <widget class="QLabel" name="label_10" >
          <property name="text" >
           <string>Dirac width (epsilon)</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" >
         <widget class="QDoubleSpinBox" name="doubleSpinBox_epsilon" />
        </item>
        <item row="4" column="0" >
         <widget class="QLabel" name="label_11" >
          <property name="text" >
           <string>Time step (delta)</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1" >
         <widget class="QDoubleSpinBox" name="doubleSpinBox_delta" />
        </item>
       </layout>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_7" >
        <property name="text" >
         <string>Regions to be segmented</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="lineEdit_regions" />
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" >
      <item>
       <widget class="QCheckBox" name="checkBox_usemodel" >
        <property name="text" >
         <string>use model file</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" >
        <item>
         <widget class="QLabel" name="label_6" >
          <property name="text" >
           <string>Model file</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_ModelFilename" />
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_file" >
          <property name="text" >
           <string>file...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3" >
      <item>
       <widget class="QPushButton" name="pushButton_cancel" >
        <property name="text" >
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_ok" >
        <property name="text" >
         <string>OK</string>
        </property>
        <property name="default" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
